S1_NRB/metadata/extract.py

```diff
@@ -14,12 +14,296 @@
+         'sentinel-1b': 'http://database.eohandbook.com/database/missionsummary.aspx?missionID=576'}[
+            meta['common']['platformFullname']]
+    meta['common']['polarisationChannels'] = sid0.polarizations
+    meta['common']['polarisationMode'] = prod_meta['pols'][0]
+    meta['common']['processingLevel'] = 'L1C'
+    meta['common']['radarBand'] = 'C'
+    meta['common']['radarCenterFreq'] = 5405000000
+    meta['common']['sensorType'] = 'RADAR'
+    meta['common']['swathIdentifier'] = swath_id
+    meta['common']['wrsLongitudeGrid'] = str(sid0.meta['orbitNumbers_rel']['start'])
+    
+    # Product metadata (sorted alphabetically)
+    meta['prod']['access'] = config['meta']['access_url']
+    meta['prod']['ancillaryData_KML'] = 'https://sentinel.esa.int/documents/247904/1955685/S2A_OPER_GIP_TILPAR_MPC__' \
+                                        '20151209T095117_V20150622T000000_21000101T000000_B00.kml'
+    meta['prod']['acquisitionType'] = 'NOMINAL'
+    meta['prod']['azimuthNumberOfLooks'] = prod_meta['ML_nAzLooks']
+    meta['prod']['backscatterConvention'] = 'linear power'
+    meta['prod']['backscatterConversionEq'] = '10*log10(DN)'
+    meta['prod']['backscatterMeasurement'] = 'gamma0' if re.search('g-lin', ref_tif) else 'sigma0'
+    meta['prod']['card4l-link'] = 'https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf'
+    meta['prod']['card4l-version'] = '5.5'
+    meta['prod']['crsEPSG'] = str(prod_meta['epsg'])
+    meta['prod']['crsWKT'] = prod_meta['wkt']
+    meta['prod']['compression_type'] = compression
+    meta['prod']['compression_zerrors'] = z_err_dict
+    meta['prod']['demEGMReference'] = DEM_MAP[config['dem_type']]['egm']
+    meta['prod']['demEGMResamplingMethod'] = 'bilinear'
+    meta['prod']['demGSD'] = DEM_MAP[config['dem_type']]['gsd']
+    meta['prod']['demName'] = config['dem_type'].replace(' II', '')
+    meta['prod']['demReference'] = DEM_MAP[config['dem_type']]['ref']
+    meta['prod']['demResamplingMethod'] = 'bilinear'
+    meta['prod']['demType'] = DEM_MAP[config['dem_type']]['type']
+    meta['prod']['demAccess'] = DEM_MAP[config['dem_type']]['access']
+    meta['prod']['doi'] = config['meta']['doi']
+    meta['prod']['ellipsoidalHeight'] = None
+    meta['prod']['speckleFilterApplied'] = False
+    meta['prod']['geoCorrAccuracyEasternBias'] = None
+    meta['prod']['geoCorrAccuracyEasternSTDev'] = None
+    meta['prod']['geoCorrAccuracyNorthernBias'] = None
+    meta['prod']['geoCorrAccuracyNorthernSTDev'] = None
+    meta['prod']['geoCorrAccuracy_rRMSE'] = \
+        calc_geolocation_accuracy(swath_identifier=swath_id, ei_tif=ei_tif[0], etad=config['etad']) \
+            if len(ei_tif) == 1 and sid0.product == 'SLC' and 'copernicus' in config['dem_type'].lower() else None
+    meta['prod']['geoCorrAccuracyReference'] = 'https://s1-nrb.readthedocs.io/en/latest/general/geoaccuracy.html'
```

Comment:
Why the latest version? The way this is calculated might change in future versions and the documentation might thus no longer apply to the product at hand.

Reply:
You are totally right! Not sure anymore what my thought process was back then. I reckon that I checked some hrefs in the metadata and noticed that this one was broken. Probably just because it was pointing to a dev version.

I have reverted this in https://github.com/SAR-ARD/S1_NRB/pull/117/commits/f026484e2af78cce743bf7a5a2fcc4e8ef4f15cb